### DISCUSSION TITLE
Should I `.gitignore` the `workspace.json` file?

### DISCUSSION THREAD

**Discussion by eamonburns** [Vinzent03/obsidian-git#709] [Q&A] [answered]
I have been getting merge conflicts somewhat regularly, and mostly in `workspace.json`. I think that it just holds state about the view into the Vault, and no actual critical data itself. Is that true? And should I `.gitignore` it?

**Comment by Vinzent03** [comment, marked as answer]
Yeah that's a good idea. See [here](https://publish.obsidian.md/git-doc/Tips-and-Tricks#Gitignore) and [here](https://publish.obsidian.md/git-doc/Common+issues#Files+in+%60.gitignore%60+aren't+ignored) for help.

**Reply by eamonburns** [reply]
Ok, thank you
